spotpy/algorithms/fast.py

```diff
@@ -139,6 +139,13 @@ def analyze(self, problem, Y, D, parnames, M=4, print_to_console=False):
 
         if Y.size % (D) == 0:
             N = int(Y.size / D)
+        elif Y.size > D:
```

Comment:
Yes, here we have to discuss, why, if I set the rep=200, I get a result with a length of 204, and then I had the idea that we somehow can handle wrong user inputs, because there are no right inputs 😕 .

Reply:
Yes, I think to provide a robust code, it is better to correct the number of repetitions (in this case 204) by default. As the `def analyse(self.,..)` function can also be used after the sampling (you can also find it in the analyser.py), I thought it would be good to test again, if a correct number of repetitions can be found in the results array.
Anyway, to answer your point: wouldnt rep=204 a "right input"?